contracts/staking/fuse/utils/StakingRewardsPerEpoch.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Add comments to all methods (see how we comment in our smart contracts)
Add inline comments explaining all calculations